.github/workflows/argus-docker-build.yaml

```diff
@@ -54,14 +54,19 @@ jobs:
       contents: read
       pull-requests: read
     steps:
-      - uses: chanzuckerberg/github-actions/.github/actions/argus-builder/build-prep@v3.4.1
+      - uses: 8BitJonny/gh-get-current-pr@3.0.0
+        id: findPr
+        with:
+          filterOutClosed: true
+          filterOutDraft: false
+      - uses: chanzuckerberg/github-actions/.github/actions/argus-builder/build-prep@rzheng/stack-label
```

Comment:
Don't forget to change this in a follow up PR

Reply:
https://github.com/chanzuckerberg/github-actions/pull/335